### PR TITLE
Remove signal legacy fields

### LINKED ISSUE #5349: Delete field legacy_signals
## Dependencies

- [x] #5172 
- [x] #5217 

## AC

- [ ] Remove all legacy fields

### PR DESCRIPTION
**Pull request by flomonster**
close #5349

### CODE REVIEW

**Review by leovalais — APPROVED** [review]
LGTM (editoast)

File: editoast/migrations/2023-10-24-071333_signal_remove_legacy_fields/down.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+UPDATE infra_object_signal SET data = jsonb_set(data, '{extensions, sncf, is_in_service}', 'false');
+UPDATE infra_object_signal SET data = jsonb_set(data, '{extensions, sncf, is_lightable}', 'false');
+UPDATE infra_object_signal SET data = jsonb_set(data, '{extensions, sncf, is_operational}', 'false');
+UPDATE infra_object_signal SET data = jsonb_set(data, '{extensions, sncf, support_type}', '""');
+UPDATE infra_object_signal SET data = jsonb_set(data, '{extensions, sncf, type_code}', '""');
+UPDATE infra_object_signal SET data = jsonb_set(data, '{extensions, sncf, value}', '""');
+
+ALTER TABLE infra ALTER column railjson_version SET DEFAULT '3.4.4';
+
+UPDATE infra SET railjson_version = '3.4.4';
```
Comment (Khoyo) [inline editoast/migrations/2023-10-24-071333_signal_remove_legacy_fields/down.sql]: Missing newline at EOF

File: editoast/migrations/2023-10-24-071333_signal_remove_legacy_fields/up.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    #- '{extensions,sncf,is_in_service}' 
+    #- '{extensions,sncf,is_lightable}' 
+    #- '{extensions,sncf,is_operational}' 
+    #- '{extensions,sncf,support_type}' 
+    #- '{extensions,sncf,type_code}' 
+    #- '{extensions,sncf,value}';
+
+ALTER TABLE infra ALTER column railjson_version SET DEFAULT '3.4.5';
+
+UPDATE infra SET railjson_version = '3.4.5';
```
Comment (Khoyo) [inline editoast/migrations/2023-10-24-071333_signal_remove_legacy_fields/up.sql]: Missing newline at EOF

File: editoast/src/converters/utils.rs [hunk 3]
```diff
@@ -440,21 +439,9 @@ fn sncf_extensions(node: &Node) -> SignalSncfExtension {
...
-        ],
-        default_aspect: "CARRE".to_string(),
-        installation_type: "CARRE".to_string(),
-        is_in_service: true,
-        is_lightable: true,
-        is_operational: true,
         label,
         side,
-        ..Default::default()
+        kp: "0+00".into(),
```
Comment (Khoyo) [inline editoast/src/converters/utils.rs]: Everywhere else we put empty strings when the KP is unknown. Should we change those too?

**Review by Khoyo — APPROVED** [review]
Otherwise the core part LGTM

**Review by Tguisnet — APPROVED** [review]
LGTM (Editoast part)

**Review by younesschrifi — APPROVED** [review]

**Pull request merged**